po/uk.po

```diff
@@ -17621,12 +17631,11 @@ msgid ""
 "host and the target through a number of ringbuffers."
 msgstr ""
 "RTT (Real Time Transfers) — це механізм передачі даних між хостом "
-"налагодження та цільовим сервером через кілька кільцевих буферів."
+"налагодження та цільовим хостом через кілька кільцевих буферів."
```

Comment:
Not quite sure if it's correct to name the [debuggee target](https://probe.rs/docs/tools/cargo-embed/#target) as a _host_...

Reply:
Fair point, I think I found a good translation equivalent, thank you